data/libraries.json

```
@@ -757,5 +757,14 @@
             "JavaScript"
         ],
         "url": "http://ivan.vucica.net"
+    },
+    {
+        "doap": "https://git.sr.ht/~nicoco/slidge/tree/master/item/doap.xml",
+        "last_renewed": "2023-01-18T00:00:00",
+        "name": "Slidge",
+        "platforms": [
+            "Python"
+        ],
+        "url": "https://sr.ht/~nicoco/slidge/"
```

```suggestion
        "url": null
```

Same goes for the url key. It's already declared in Slidge's DOAP file.